### DISCUSSION TITLE
user-guide/plugins/zotmoov

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#310] [文档评论区]
# user-guide/plugins/zotmoov

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/plugins/zotmoov

**Comment by hubahu** [comment]
咨询一下与attanger的区别以及哪个更方便呢

**Reply by baozhenzhou** [reply]
亲测这个更方便

**Comment by swaiwai-ss** [comment]
使用方法第二步，「已链接附件的根目录」选项设置同一个文件夹时，弹出 "您无法在 Zotero 数据目录中选择一个关联附件的根目录" 这是为什么？应该怎么设置？

**Reply by swaiwai-ss** [reply]
![屏幕截图 2025-03-26 091400](https://github.com/user-attachments/assets/689fc1bb-60a6-4a4e-9b26-9924a77aa5f2)
附图

**Comment by colin-creator** [comment]
如果需要在多台设备之间同步，打开「设置」 -> 「高级」，将「文件和文件夹」 -> 「已链接附件的根目录」选项也设置为这个目录。

你好，我已经按照这个设置了，为什么还是无法多设备同步呢？坚果云也配置成功啦
